components/cluster/service-hacluster-remote/files/svc-hacluster-remote

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/bash
+# Start/stop HACluster service
+#
+. /lib/svc/share/smf_include.sh
+
+#==#==# Pacemaker Remote
+# Use a custom directory for finding the authkey.
+PCMK_authkey_location=/etc/pacemaker/authkey
+#
+# Specify a custom port for Pacemaker Remote connections
+PCMK_remote_port=3121
+
+## Tracing with ebug version
+# PCMK_trace_files=1
+# PCMK_trace_functions=1
+# PCMK_trace_formats=1
+# PCMK_trace_tags=1
+
+export PCMK_ipc_type=socket
+export PCMK_logfile=/var/log/hacluster/pacemaker.log
+
+export CLUSTER_USER=hacluster
+export CLUSTER_GROUP=haclient
+export PATH=/usr/gnu/bin:${PATH}
+MYNAME=hacluster
+APPPATH=/usr/sbin/
+PACEMAKERD=pacemaker_remoted
+PACEMAKER_PROCESSES=pacemaker
+SLEEPINTERVALL=5
+
+
+# all processes should log to the common folder /var/log/${MYNAME
+[ ! -d /var/log/${MYNAME} ] && mkdir -p /var/log/${MYNAME}
+
+# dumps of crashs, check the folder for problems
+[ ! -d /var/lib/pacemaker/cores ] && mkdir -p /var/lib/pacemaker/cores
+
+touch /var/log/${MYNAME}/${MYNAME}.log
+chown ${CLUSTER_USER}:${CLUSTER_GROUP} /var/log/${MYNAME}/${MYNAME}.log
+[ ! -d /var/run/${MYNAME} ] && mkdir -p /var/run/${MYNAME}
+chown -R ${CLUSTER_USER}:${CLUSTER_GROUP} /var/run/${MYNAME}
+
+killapp() {
+        pid=`pgrep -f $1`
+        if [ "x${pid}" != "x" ]; then
+                kill -9 ${pid}
+        fi
+        return 0
+}
+
+start() {
+        stop
```

Comment:
Drop this as SMF has :kill builtin and that has better results. Also, doing pkill et al from SMF manifest is not the best practice.